java/java.lsp.server/vscode/src/extension.ts

```diff
@@ -75,6 +75,7 @@ export let client: Promise<NbLanguageClient>;
 export let clientRuntimeJDK : string | null = null;
 export const MINIMAL_JDK_VERSION = 17;
 export const TEST_PROGRESS_EVENT: string = "testProgress";
+export let debugConsoleListeners: any[] = [];
```

Comment:
Please type the callback as a `function(output : string) : void` or something != any. Right now the code requires the `callback` member to be present.